test/iron-ajax.html

```diff
@@ -216,13 +215,16 @@
           expect(requests).to.deep.eql(ajax.activeRequests);
         });
 
-        test('empties `activeRequests` when requests are completed', function(done) {
+        test('empties `activeRequests` when requests are completed', function() {
           server.respond();
-          timePasses(1).then(function() {
+          return requests[0].completes.then(function() {
```

Comment:
Would it be possible to use `Promise.all` here to avoid a long chain of `then` calls?